services/core-api/app/api/commons/resources/email_resource.py

```diff
@@ -0,0 +1,25 @@
+from app.api.services.email_service import EmailService
+from flask_restplus import Resource
+from flask import request
+
+from app.api.utils.access_decorators import (requires_any_of, VIEW_ALL)
+from app.api.utils.resources_mixins import UserMixin
+from app.api.exception.mds_core_api_exceptions import MDSCoreAPIException
+
+class EmailResource(Resource, UserMixin):
```

Comment:
I would suggest that we maybe do not expose this as a generic way of sending emails accross the app. I've got some concerns with this in terms of potential abuse, as this would in essence allow anyone with access to send arbitrary emails on behalf of the mds team. The other concern I have would be that we allow plain html as input which would also be vulnerable (e.g. a user could easily add javascript to the html).

As an alternative we could make this a specific "Report Error" endpoint that takes in the required fields so we can do some basic validation on it and create an email template in the templates folder which would escape what's passed in.

Reply:
Thanks Simen,
Added an endpoint for report-error, and only the error details are accepted in that. And email get sent with user info from BE. And the html template is used for the email content.